.github/ISSUE_TEMPLATE/2-deprecate-sgid-dataset.md

```diff
@@ -30,15 +30,19 @@ The replaced data is still accessible via our shelved policy in AGOL (a link to
 1. _~Strike~ out all items that do not apply._
 
 - [ ] Remove data from the Internal SGID (name, completed: `2021/00/00`)
+- [ ] Mark ArcGIS Online item as deprecated in preparation for future deletion (name, completed: `2021/00/00`)
+  - [ ] Add `Deprecated` to `AGOL_PUBLISHED_NAME` in `SGID.META.AGOLItems` (name, completed: `2021/00/00`)
+  - [ ] Change `Authoritative` field to `d` in `SGID.META.AGOLItems` to set the `Deprecated` AGOL flag (name, completed: `2021/00/00`)
+  - [ ] Add note to ArcGIS Online description noting when layer will be deleted and any replacement layer (name, completed: `2021/00/00`)
 - [ ] Delete ArcGIS Online item (name, completed: `2021/00/00`)
-  - [ ] Add `deprecated` tag to ArcGIS Online item (name, completed: `2021/00/00`)
-  - [ ] Add `Deprecated` to ArcGIS Online item title(name, completed: `2021/00/00`)
 - [ ] Unshare item from Open Data (name, completed: `2021/00/00`)
+  - currently achieved by either deleting the AGOL item, changing the `AGOL_ITEM_ID` field in `SGID.META.AGOLItems` to something other than an Item ID and manually removing the item from all SGID groups, or shelving the data.
```

Comment:
**thought**: will these instructions be simpler to follow as separate checkboxes?

Reply:
The Open Data sharing is a little more nuanced/tightly coupled than I think we realized earlier. See https://github.com/agrc/auditor/issues/52 for my thoughts on this. I felt it important to at least get something in here to note what was going on (my other thought was to delete this row completely because we don't have a manual way of unsharing it with Open Data without affecting other things)

Reply:
If the `Authoritative` field was set to `d`, could Auditor or some other process use that to update the `AGOL_ITEM_ID` to unshare it from Open Data?

Reply:
That could be a consideration for after we sort out the Open Data sharing tangle-up, yes.